src/assets/icon-button.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
```

Comment:
I should have noticed this before. The `assets` folder should just be for assets -- it is copied to the deploy folder. This should be in `src/components/utilities` I would think.

Reply:
This has been moved to `src/components/utilities`!